tests/test_pluggable.py

```diff
@@ -652,9 +654,35 @@ def test_retrieve_subject_token_missing_expiration_time(self):
                 _ = credentials.retrieve_subject_token(None)
 
             assert excinfo.match(
-                r"The executable response is missing the expiration_time field."
+                r"The output file response must contain the expiration_time field when success=true."
             )
 
+    @mock.patch.dict(os.environ, {"GOOGLE_EXTERNAL_ACCOUNT_ALLOW_EXECUTABLES": "1"})
```

Comment:
Add another test that validates that an error is thrown when reading from the output_file and expiration_time is missing